src/server/game/Entities/Creature/Creature.h

```diff
@@ -372,6 +372,8 @@ class TC_GAME_API Creature : public Unit, public GridObject<Creature>, public Ma
 
         std::string GetDebugInfo() const override;
 
+        void ExitVehicle(Position const* exitPosition = nullptr);
```

Comment:
```cpp
/src/server/game/Entities/Creature/Creature.h:375:14:
 fatal error: 'ExitVehicle' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
        void ExitVehicle(Position const* exitPosition = nullptr);
             ^
```

Reply:
Ups, looks like Visual Studio isn't capable of that.
https://developercommunity.visualstudio.com/t/add-a-warning-if-override-keyword-is-not-used-in-d/396484
I fixed it.